backend/src/database/dto.rs

```diff
@@ -0,0 +1,80 @@
+use chrono::NaiveDateTime;
+use serde::{Deserialize, Serialize};
+use sqlx::FromRow;
+use uuid::Uuid;
+
+pub type Hash = String;
+
+#[derive(Serialize, Deserialize)]
+pub struct Claim {
+    #[serde(rename = "cTypeHash")]
+    pub ctype_hash: String,
+    contents: serde_json::Value,
+    pub owner: String,
+}
+
+#[derive(Serialize, Deserialize)]
+pub struct Credential {
+    pub claim: Claim,
+    #[serde(rename = "claimNonceMap")]
+    claim_nonce_map: serde_json::Value,
+    #[serde(rename = "claimHashes")]
+    claim_hashes: Vec<Hash>,
+    #[serde(rename = "delegationId")]
+    delegation_id: Option<String>,
+    legitimations: Vec<Credential>,
+    #[serde(rename = "rootHash")]
+    pub root_hash: Hash,
+}
+
+#[derive(Serialize, Deserialize, FromRow)]
+pub struct AttestationResponse {
+    pub id: Uuid,
+    pub approved: bool,
+    pub revoked: bool,
+    pub created_at: NaiveDateTime,
+    pub deleted_at: Option<NaiveDateTime>,
```

Comment:
technically there is also revoked_at that is distinct from deleted_at

Reply:
True, I have also added a field `approved_at` in: [9c2cbb7](https://github.com/KILTprotocol/attester/pull/1/commits/9c2cbb72ea85117c69eaaf66e9f49988a3cef07c)